packages/editor/src/components/parts/web-service/WebServicePart.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { DeepPartial, render, renderHook, screen } from 'test-utils';
+import { ElementData, InscriptionValidation, WebserviceStartData } from '@axonivy/inscription-protocol';
+import { PartStateFlag } from '../../editors';
+import { useWebServicePart } from './WebServicePart';
+
+const Part = () => {
+  const part = useWebServicePart();
+  return <>{part.content}</>;
+};
+
+describe('WebServicePart', () => {
+  function renderPart(data?: WebserviceStartData) {
+    render(<Part />, {
+      wrapperProps: { data: data && { config: data } }
+    });
+  }
+
+  test('empty data', async () => {
+    renderPart();
+    expect(screen.queryByText('Permission')).toBeInTheDocument();
+    expect(screen.queryByText('Exception')).toBeInTheDocument();
+  });
+
+  test('full data', async () => {
+    renderPart({
+      permission: {
+        anonymous: false,
+        error: '>> Ignore Exception',
+        role: 'Test'
+      },
+      exception: {
+        condition: '0===0',
+        enabled: true,
+        message: 'hallo'
+      }
+    });
+    expect(screen.queryByText('Permission')).toBeInTheDocument();
+    expect(screen.queryByText('Exception')).toBeInTheDocument();
+  });
+
+  function assertState(expectedState: PartStateFlag, data?: DeepPartial<WebserviceStartData>, validation?: InscriptionValidation) {
+    const { result } = renderHook(() => useWebServicePart(), {
+      wrapperProps: { data: data && { config: data }, validations: validation && [validation] }
+    });
+    expect(result.current.state.state).toEqual(expectedState);
+  }
+
+  test('configured', async () => {
+    assertState('empty');
+    assertState('configured', {
+      permission: {
+        error: '>> Ignore Exception'
+      },
+      exception: {
+        condition: '0===0'
+      }
+    });
+
+    assertState('error', undefined, { path: 'permission.cause', message: '', severity: 'ERROR' });
+    assertState('warning', undefined, { path: 'permission.error', message: '', severity: 'WARNING' });
+    assertState('error', undefined, { path: 'exception.cause', message: '', severity: 'ERROR' });
+    assertState('warning', undefined, { path: 'exception.error', message: '', severity: 'WARNING' });
+  });
+
+  test('reset', () => {
+    let data: DeepPartial<ElementData> = {
+      config: {
+        exception: {
+          message: 'hallo',
+          enabled: true
+        }
+      }
+    };
+    const view = renderHook(() => useWebServicePart(), {
+      wrapperProps: { data, setData: newData => (data = newData) }
+    });
+    expect(view.result.current.reset.dirty).toEqual(true);
+
+    view.result.current.reset.action();
+    expect(data.config?.exception?.message).toEqual(undefined);
+    expect(data.config?.exception?.enabled).not.toBeChecked;

```

Comment:
to be false or true
toBeChecked is something for a checkbox, but we compare here a plain boolean value